lib/yum.sh

```diff
@@ -83,7 +83,11 @@ yum_R() {
 }
 
 yum_Si() {
-  yum info "$@"
+  if ! command -v repoquery > /dev/null 2>&1; then
```

Comment:
I forgot to mention we have this DRY method: `_require_programs` that you see on https://github.com/icy/pacapt/blob/2e61bcfb073e3a060824d8df0549a8f0d0bab107/lib/00_core.sh#L48 . Can we start to use them? Thanks a lot.